<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="board_cards">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>board cards</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pot_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pot</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bot_hand_display">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bot hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bot_stack_display">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bot stack</string>
   </property>
  </widget>
  <widget class="QLabel" name="P_hand_display">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="P_stack_display">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P stack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Fold_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Call_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Call</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Raise_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Raise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="All_in_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All in</string>
   </property>
  </widget>
  <widget class="QLabel" name="Message_display">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>                     playing HUTH Poker</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_call_display">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>to call: 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
